### ISSUE TITLE
SelectOneMenu: input element missing aria tags

### ISSUE THREAD

**Issue by mrinalkumari06** [primefaces/primefaces#10034]
### Describe the bug

**Problem:**
On primefaces 11.0.11, 
1. When navigating on dropdown using keyboard , it works fine.
2. When JAWS (Job Access With Speech) tool is open, and navigate through dropdown, its not happening and so JAWS is not announcing the option value.
Same case worked fine with primefaces 8.0.2 and recently we did migration to primefaces 11.0.11.

When compared one menu HTML of Primefaces 8.0.2 and 12 with Primefaces 11.0.11, found that input element missing 'aria tags: aria-labelledby, aria-owns = "option ul" ' 
I am not sure if this is the root cause , but this is the difference we found. Can these elements put back in primefaces 11?

### Expected behavior

with JAWS open, keyboard navigation should work on dropdown.

If it work then JAWS will able to announce the value.

### PrimeFaces edition

Community

### PrimeFaces version

11.0.11

### JSF implementation

Mojarra

### JSF version

2.0

### Java version

JDK 17

### Browser(s)

Chrome

**Comment by jepsar** [comment]
Can you please try with 13?

**Comment by melloware** [comment]
It sounds like @mrinalkumari06 you are saying it works fine on 12.0.0 just not on 11.0.11?  If so please read: https://github.com/primefaces/primefaces#community--elite--pro

This tracker is for community issues only if you have Elite issues you would need to open a ticket through a PRO account if you have one.

**Comment by melloware** [comment]
Also SelectOne ARIA support has been completely redone for 13.0.0 and verified see: https://github.com/primefaces/primefaces/issues/9279

**Comment by melloware** [comment]
No response from OP.  Will close until we hear back differently.
